pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -620,6 +621,20 @@ def _render_jira_template(fields: Dict, template_vars: Dict):
             _LOGGER.warning("Failed to render JIRA template text: %s", ex)
         return fields
 
+    def set_advisory_dependencies(self, advisories):
+        target_kind = ['rpm', 'metadata']
+        expected_blocking = {advisories['image'], advisories['metadata']}
+        _LOGGER.info("Setting advisory dependencies for %s", target_kind)
+        for kind in target_kind:
+            target_advisory_id = advisories[kind]
+            blocking: Optional[List] = get_blocking_advisories(target_advisory_id)
+            if not blocking:
+                raise ValueError(f"Failed to fetch blocking advisories for {target_advisory_id} ")
+            if expected_blocking.issubset(set(blocking)):
+                continue
+            for blocking_advisory_id in expected_blocking:
+                set_blocking_advisory(target_advisory_id, blocking_advisory_id, "SHIPPED_LIVE")
```

Comment:
blocking_advisory or target_advisory could also be DROPPED. I think we need to skip those advisories too.

Reply:
This might be a problem.. we usually drop advisories after prepare-release completes, so by this time the dependency would already be set

Reply:
This could be a logic for drop advisory job

Reply:
Right now I am okay with this failing and displaying a warning

Reply:
A slack message will also be sent

Reply:
Sounds good to me.